products/sle/main.pm

```diff
@@ -442,6 +445,9 @@ sub load_inst_tests() {
                 loadtest "installation/user_settings.pm";
             }    # sles4sap wizard installation doesn't have user_settings step
         }
+        elsif (get_var('IMPORT_USER_DATA')) {
+            loadtest 'installation/user_import.pm';
+        }
```

Comment:
You suggested optionally importing user data in the progress ticket, so that is what it does. I did not find any existing test for importing user data so created a new one.